src/lib/core/components/visualizations/Visualizations.scss

```diff
@@ -298,3 +298,23 @@ $data-table-inner-border: 1px solid #888;
     border-bottom: $data-table-inner-border;
   }
 }
+
+// boxplot stat table for alphadiv and abundance
```

Comment:
We should ask clinepi if they want the stats table for their boxplot, too. Would you mind demoing at the front end meeting tomorrow morning? I'll reach out to the clinepi folks

Reply:
I can demo if you want, for sure 😃